_posts/2021-08-14-eslint-v8.0.0-beta.0-released.md

```diff
@@ -4,6 +4,7 @@ title: ESLint v8.0.0-beta.0 released
 tags:
   - release
   - major
+author: Brandon Mills
```

Comment:
Rather than the names, I’d prefer to use GitHub user names and then use a [data file](https://www.11ty.dev/docs/data-global/) to look up the full name. That way we can also link to their GitHub profile easily.

Reply:
I can change the names to their GitHub user names. For the data file do we need a new one created or can we use the one that is already created in `team.json` in the `_data` directory.

Reply:
Ah good point! For now, the team.JSON file will work. 👍

Reply:
Just double checked this: the team.JSON file won’t work due to the format it’s in. It’s top level properties are categories so you can’t look up individual people. So, a new data file is the simplest way to go.

Reply:
Ok, sounds good. I will create a new data file.

Reply:
@nzakas There are blog posts that say they are autogenerated by ESLint Jenkins. Should I leave those blank or should I put ESlint Jenkins? Also what should I do with the articles that a couple of authors, should I put all the authors?

Reply:
You can leave the Jenkins posts with a blank author.

For multiple authors, maybe use `authors` in the frontmatter and specify an array? Then in the template you can check for `authors` vs. `author` to determine whether there are multiple authors. If there are multiple authors, I think you can just put them on the same line separated by commas.

Reply:
Ok, thank you.